src/python/turicreate/data_structures/sframe.py

```diff
@@ -2188,7 +2188,42 @@ def _imagecols_to_stringcols(self, num_rows=10):
                 if t in image_column_names:
                     printed_sf[t] = self[t].astype(str)
         return printed_sf.head(num_rows)
+    
+    def drop_duplicates(self, subset):
+        """
+        Return SFrame with duplicate rows removed, optionally only considering certain columns.
+
+        Parameters
+        ----------
+        subset : column label or sequence of labels, optional
```

Comment:
This parameter isn't actually optional. You need to specify it. It doesn't by default use all the columns.